src/utils/prices.py

```diff
@@ -29,33 +29,39 @@ class TokenId(Enum):
     COW = "cow-cow-protocol-token"
     USDC = "usdc-usd-coin"
 
+    def decimals(self) -> int:
+        """Decimals for each of the token variants."""
+        if self == TokenId.USDC:
+            return 6
+        return 18
```

Comment:
The only relevant tokens here have 18 decimals and USDC has 6. Realistically USDC serves no purpose in this project and could be removed. The only tokens that matter at all here are COW and ETH. However, in the future this may change (if we compute all token imbalances internally).